content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -525,11 +525,13 @@ This means that support is still available but will be removed in a future Kuber
 Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  (**not available** starting v1.27)
 * [`azureDisk`](/docs/concepts/storage/volumes/#azuredisk) - Azure Disk
-  (**not available** in v1.27)
+  (**not available** starting v1.27)
 * [`cinder`](/docs/concepts/storage/volumes/#cinder) - Cinder (OpenStack block storage)
-  (**not available** in v1.26)
+  (**not available** starting v1.26)
+* [`gcePersistentDisk`](/docs/concepts/storage/volumes/#gcepersistentdisk) - GCE Persistent Disk
+  (**not available** starting v1.28)
```

Comment:
How do we confidently know which release removed this in-tree driver?

Reply:
![Screenshot 2023-12-14 at 1 15 49 PM](https://github.com/kubernetes/website/assets/1185309/6ae33458-bd4d-40e9-9a9c-d0e0f4b23651)
Here's a screenshot from SIG Storage session at KubeCon NA this year. The in-tree GCE PD plugin was removed in 1.28.

Reply:
Thank you @xing-yang